sdk/include/DiscordRPC/discord_register.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
```

Comment:
@DenysOliinyk am I right that these 2 files (`discord_register.h` and `discord_rpc.`) are from Discord itself?

Reply:
@collectioner, yes, they are.

Reply:
@collectioner yup, from their official repo

Reply:
Thanks you both, I resolved the comment.